deployment_metadata/deployment_metadata.go

```diff
@@ -0,0 +1,64 @@
+package deployment_metadata
```

Comment:
Packages shouldn't have underscores on them according to [Go guidelines](https://golang.org/doc/effective_go#package-names).